lms/djangoapps/learner_dashboard/views.py

```diff
@@ -61,6 +62,9 @@ def program_details(request, program_id):
 
     program_data = utils.ProgramDataExtender(program_data, request.user).extend()
 
+    # user preferences (language and timezone) sent for dateutils in course_card_model.js
+    program_data['user_preferences'] = get_user_preferences(request.user)
+
```

Comment:
I know you don't want to go back and forth on this much further, but: adding it as a separate variable in the template context is cleaner. As designed, `program_data` is only meant to hold program data (e.g., program title, link to the about page, included course runs). User preferences are program-independent, just like the URLs included separately in the context.